### ISSUE TITLE
Navigation entry with spaces creates multiple directories in the temporary directory 

### ISSUE THREAD

**Issue by froschdesign** [jdoiro3/mkdocs-multirepo-plugin#10]
A navigation entry with a space in the title does not work. 

```yaml
nav:
    - "Imported Repo": '!import https://github.com/…'
```

CLI output:

```bash
[Errno 2] No such file or directory: PosixPath('/path/to/project/temp_dir/Imported Repo')
```

This results in two directories in the temporary directory: 

* `temp_dir`
  * `Imported`
    * `docs`
      * …
  * `Repo`

Each individual word creates a new directory, this means 4 directories for "Imported Repo With Spaces".

**Comment by jdoiro3** [comment]
This bug should be resolved in `v0.3.1` and was tested. @froschdesign, can you confirm this on your end?

**Comment by froschdesign** [comment]
It works! 👍
